docs/development/postman/tractusx-dtr-aas-3.0.0-collection.json

```diff
@@ -25,7 +25,7 @@
 								}
 							}
 						},
-						"url": "{{baseUrl}}/api/v3.0/shell-descriptors"
```

Comment:
Why the change in collection 3.0.0 needed?

Reply:
I thought if I do not change this, then it's not valid anymore. But since we have new collection, I will revert the changes in this.